### ISSUE TITLE
Use validation set to decide when to stop training

### ISSUE THREAD

**Issue by lene** [lene/style-scout#25]
https://stackoverflow.com/questions/46308374/what-is-validation-data-used-for-in-a-keras-sequential-model gives some pointers how it is used
